content/sensu-go/5.10/guides/scale-event-storage.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Scale Sensu Go with Enterprise Datastore"
+linkTitle: "Routing Alerts with Filters"
```

Comment:
Is this link title correct? Should be something like "Scaling with Enterprise datastore" instead?

Reply:
```suggestion
linkTitle: "Scaling with Enterprise datastore"
```